doc/start/os-recommendations.rst

```diff
@@ -14,12 +14,12 @@ Linux Kernel
 - **Ceph Kernel Client**
 
   If you are using the kernel client to map RBD block devices or mount
-  CephFS, the general advice is to use a "stable" or "longterm
+  CephFS, the general advice is to use a "stable" or "long term
   maintenance" kernel series provided by either http://kernel.org or
   your Linux distribution on any client hosts.
 
   For RBD, if you choose to *track* long-term kernels, we currently recommend
-  4.x-based "longterm maintenance" kernel series or later:
+  4.x-based "long term maintenance" kernel series or later:
```

Comment:
https://www.kernel.org/category/releases.html

Reply:
You would rather use the term coined by the Linux Kernel Organization then, even if it's incorrect - I cannot find "longterm" in any dictionary. It's OK, I'll revert that. Besides, it appears "long-term" would be the correct word here, not "long term":
* [Which is correct: long-term or long term?](https://whatis.techtarget.com/feature/Which-is-correct-long-term-or-long-term)
* [Is it long term or long-term?](https://www.future-perfect.co.uk/grammar-tips/easy-mistakes/is-it-long-term-or-long-term)
* Merriam-Webster entry for [long-term](https://www.merriam-webster.com/dictionary/long-term)